dora/core/server/master/src/main/java/alluxio/master/job/DoraLoadJob.java

```diff
@@ -497,53 +469,58 @@ public long getDurationInSec() {
   public boolean processResponse(DoraLoadTask doraLoadTask) {
     try {
       long totalLoadedBytes = doraLoadTask.getFilesToLoad().stream()
-          .map(URIStatus::getLength)
+          .map((it) -> (it instanceof UfsFileStatus ? it.asUfsFileStatus().getContentLength() : 0))
           .reduce(Long::sum)
           .orElse(0L);
       // what if timeout ? job needs to proactively check or task needs to be aware
       LoadFileResponse response = doraLoadTask.getResponseFuture().get();
       if (response.getStatus() != TaskStatus.SUCCESS) {
         LOG.debug(format("Get failure from worker:%s, failed files:%s",
-            doraLoadTask.getMyRunningWorker(), response.getFilesList()));
-        for (FileFailure fileFailure : response.getFilesList()) {
-          totalLoadedBytes -= fileFailure.getFile().getLength();
-          if (!isHealthy() || !fileFailure.getRetryable() || !addFilesToRetry(
-              fileFailure.getFile().getAlluxioPath())) {
-            addFileFailure(fileFailure.getFile().getAlluxioPath(), fileFailure.getMessage(),
-                fileFailure.getCode());
+            doraLoadTask.getMyRunningWorker(), response.getFailuresList()));
+        for (LoadFileFailure failure : response.getFailuresList()) {
+          totalLoadedBytes -= failure.getUfsStatus().getUfsFileStatus().getContentLength();
+          if (!isHealthy() || !failure.getRetryable() || !addFilesToRetry(
+              failure.getUfsStatus().getUfsFullPath())) {
+            addFileFailure(
+                mLoadRootAlluxioUri.join(failure.getUfsStatus().getName()).toString(),
```

Comment:
i'm a little doubtful of the concatenation here, feel like we can just pass UfsFullPath